src/Directory.Packages.props

```diff
@@ -0,0 +1,21 @@
+<Project>
+  <PropertyGroup>
+    <ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageVersion Include="Antlr4" Version="4.6.6" />
+    <PackageVersion Include="Antlr4.Runtime" Version="4.6.6" />
+    <PackageVersion Include="BenchmarkDotNet" Version="0.13.8" />
+    <PackageVersion Include="Microsoft.CSharp" Version="4.7.0" />
+    <PackageVersion Include="Microsoft.CodeAnalysis.CSharp" Version="4.7.0" />
+    <PackageVersion Include="Microsoft.CodeAnalysis.Common" Version="4.7.0" />
+    <PackageVersion Include="Microsoft.NET.Test.Sdk" Version="17.7.2" />
```

Comment:
.NET 6 SDK has the Test Tool (dotnet test) v17.3.3. Wondering whether this doesn't work well with this package?

Reply:
Build and test passes, so should be fine